src/App.js

```diff
@@ -1,232 +1,47 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-    return (
-      <div className="App">
-     
-    <nav className="navbar navbar-inverse navbar-fixed-top" role="navigation">
-        <div className="container">
-          
-            <div className="navbar-header">
-                <button type="button" className="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
-                    <span className="sr-only">Toggle navigation</span>
-                    <span className="icon-bar"></span>
-                    <span className="icon-bar"></span>
-                    <span className="icon-bar"></span>
-                </button>
-                <a className="navbar-brand" href="#">Start Bootstrap</a>
-            </div>
-          
-            <div className="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul className="nav navbar-nav">
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Services</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                </ul>
-            </div>
-         
-        </div>
-      
-    </nav>
-
- 
-    <div className="container">
-
+import React from "react";
+import Header from "./components/Header";
+import Carousel from "./components/Carousel";
+import ProductDetail from "./components/ProductDetail";
+import Footer from "./components/Footer";
+
+function App(props) {
+  const products = props.products;
+  console.log(products);
+
+  const productDetails = products.map((product) => {
```

Comment:
But this isn't working. Can you figure out why. What are you missing from index.js ?